src/osvr/exportheader.cmake.in

```diff
@@ -0,0 +1,89 @@
+
```

Comment:
Needs file header with license, etc - copy from another one, make sure to note that it's a generated file.